paymentsheet/api/paymentsheet.api

```diff
@@ -268,6 +268,236 @@ public abstract interface class com/stripe/android/customersheet/SetupIntentClie
 	public abstract fun provideSetupIntentClientSecret (Ljava/lang/String;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 }
 
+public final class com/stripe/android/link/LinkActivityContract$Companion {
```

Comment:
There are a lot of things here that can be marked `internal` to remove from the diff.